packages/z3-static/z3-static.4.8.13/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+maintainer: "simon@aestheticintegration.com"
+authors: "MSR"
+homepage: "https://github.com/Z3prover/z3"
+bug-reports: "https://github.com/Z3prover/z3/issues"
+license: "MIT"
+dev-repo: "git+https://github.com/Z3prover/z3.git"
+build: [
+  [ "python2.7" "scripts/mk_make.py" "--ml" "--staticlib" ]
+  [ make "-C" "build" "-j" jobs ]
+]
+
+install: [
+  [ "sh" "-c" "ocamlfind install z3 build/api/ml/* build/libz3-static.a"]
+  [ "cp" "build/z3" "%{bin}%/z3-static"]
+]
+
+depends: [
+  "ocaml"
+  "ocamlfind" {build}
+  "zarith"
+  "conf-gmp"
+  "conf-python-2-7" {build}
+]
+x-ci-accept-failures: [
+  "centos-7"  "oraclelinux-7" # C compiler is too old
+  "oraclelinux-8" # Does not ship python 2.7
+]
+synopsis: "Z3 solver"
```

Comment:
```suggestion
available: os != "macos"
synopsis: "Z3 solver"
```

Reply:
Thanks, that's a better solution than what I was trying before.